lib/dependencies/CssImportDependency.js

```diff
@@ -64,6 +68,28 @@ CssImportDependency.Template = class CssImportDependencyTemplate extends (
 		const dep = /** @type {CssImportDependency} */ (dependency);
 
 		source.replace(dep.range[0], dep.range[1] - 1, "");
+
+		if (
+			// Do nothing more for external modules
+			/^(\/\/|https?:\/\/|std:)/.test(dep.request) ||
```

Comment:
We can handle externals styles

Reply:
External styles are already considered as externals assets and so, no need to transform them. It's done there: https://github.com/webpack/webpack/blob/main/lib/WebpackOptionsApply.js#L119-L163.

Reply:
I mean https://webpack.js.org/configuration/experiments/#experimentsbuildhttp, we can bundle `http`/`https` CSS

Reply:
Oh yes, I forgot that. Thanks, I will check and update the PR.

Reply:
@alexander-akait The code is updated with `buildHttp` experiment and also with assets replacements.